requirements-dev.txt

```diff
@@ -5,9 +5,11 @@ ruff-lsp==0.0.39
 mypy==1.5.1
 pytest==7.4.2
 pytest-asyncio==0.23.2
+pytest-xdist==3.5.0
 httpx==0.25.0
 trio==0.22.2
 mkdocs-material==9.5.3
 pymdown-extensions==10.3
 mkdocs-glightbox==0.3.4
 types-PyYAML==6.0.12.12
+typer==0.9.0
```

Comment:
Where is this used?

Reply:
It's used within the AlignScore app that ends up inside the container. Theoretically you don't need this since it is installed when you build the container. But it is nice to have it in your dev environment for mypy typing